### PR TITLE
Feature pin song and test admin

### PR DESCRIPTION
**Pull request by vovanbi**
## Related Tickets:
- [#38208](https://edu-redmine.sun-asterisk.vn/issues/38208)

## WHAT this PR does?

- pin hot song and tesst admin

### CODE REVIEW

**Comment by vovanbi** [comment]
ready

File: app/Http/Controllers/Admin/AlbumController.php [hunk 1]
```diff
@@ -67,12 +68,38 @@ public function update(AlbumRequest $request, $album)
...
+        }
+    }
+
+    public function action($action, $id)
+    {
+        try {
+            $album = Album::findOrFail($id);
+            switch ($action) {
+                case 'hot':
+                    $album->hot = $album->hot ? 0 : 1;
```
Comment (quoctd-0607) [inline app/Http/Controllers/Admin/AlbumController.php]: setting cho 0, 1 nhé

File: resources/views/admin/album/index.blade.php [hunk 2]
```diff
@@ -43,8 +43,12 @@
                                 <td>
                                     <img id="album-image" src="storage/{{ $album->image }}" alt="" />
                                 </td>
-                                <td>{{ $album->hot }}</td>
-                                <td> 
+                                <td>
+                                <a href="{{ route('albums.action',['hot',$album->id]) }}" class="btn {{$album->hot == 1 ? 'btn-danger' :'btn-success'}}">
+                                {{ $album->hot == 1 ? 'hot' :  'not' }}
```
Comment (quoctd-0607) [inline resources/views/admin/album/index.blade.php]: tương tự

**Comment by vovanbi** [comment]
ready

**Pull request merged**